apm_client/README.md

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
You may provide some details here : required OS, and requirements. The goal is that anyone without any knowledge must be able to run it by simply reading the readme